src/gluonts/model/deepar/_estimator.py

```diff
@@ -134,6 +141,7 @@ def __init__(
         lags_seq: Optional[List[int]] = None,
         time_features: Optional[List[TimeFeature]] = None,
         num_parallel_samples: int = 100,
+        imputation_method: MissingValueImputation = None,
```

Comment:
This should `Optional[MissingValueImputation]` then.

Reply:
True, thanks!